pg_metadata/install/sql/upgrade/upgrade_to_1.2.0.sql

```diff
@@ -419,4 +419,36 @@ $$;
 COMMENT ON FUNCTION pgmetadata.get_dataset_item_html_content(_table_schema text, _table_name text, _locale text) IS 'Generate the metadata HTML content for the given table and given language or NULL if no templates are stored in the pgmetadata.html_template table.';
 
 
+-- capitalise existing German translations
+
+CREATE TABLE pgmetadata.t_glossary (field text, code text, label_de text, description_de text);
+INSERT INTO pgmetadata.t_glossary (field, code, label_de, description_de)
+VALUES
+('dataset.categories', 'BOU', 'Grenzen', 'Gesetzlich festgelegte Grenzen'),
+('dataset.categories', 'STR', 'Bauwerke', 'Anthropogene Bauten'),
+('dataset.categories', 'GEO', 'Geowissenschaften', 'Geowissenschaftliche Informationen'),
+('dataset.categories', 'SOC', 'Gesellschaft', 'Kulturelle und gesellschaftliche Merkmale'),
+('dataset.categories', 'ECO', 'Wirtschaft', 'Wirtschaftliche Aktivitäten, Verhältnisse und Beschäftigung'),
+('link.type', 'file', 'Eine Datei', 'CKAN Metadata Vocabulary, um die Typattribute einer CKAN-Ressource zu füllen; zeigt an, dass ein http:GET dieses URL einen Bitstream liefern sollte'),
+('dataset.confidentiality', 'OPE', 'Offen', 'Keine Einschränkungen des Zugriffs auf diese Daten'),
+('dataset.confidentiality', 'RES', 'Eingeschränkt', 'Der Zugriff auf die Daten ist auf ausgewählte Nutzer beschränkt'),
+('dataset.publication_frequency', 'DAY', 'Täglich', 'Daten werden täglich aktualisiert'),
+('dataset.publication_frequency', 'MON', 'Monatlich', 'Daten werden monatlich aktualisiert'),
+('dataset.publication_frequency', 'YEA', 'Jährlich', 'Daten werden jährlich aktualisiert'),
+('dataset.publication_frequency', 'NEC', 'Bei Bedarf', 'Daten werden bei Bedarf aktualisiert'),
+('dataset.publication_frequency', 'WEE', 'Wöchentlich', 'Daten werden wöchentlich aktualisiert'),
+('dataset.publication_frequency', 'BIA', 'Halbjährlich', 'Daten werden halbjährlich aktualisiert'),
+('dataset.publication_frequency', 'IRR', 'Unregelmäßig', 'Daten werden unregelmäßig aktualisiert'),
+('dataset.publication_frequency', 'NOP', 'Nicht geplant', 'Eine Aktualisierung der Daten ist nicht geplant'),
+('dataset.license', 'proj', 'Nur für Projektbearbeitung', NULL)
+ON CONFLICT DO NOTHING;
+
+UPDATE pgmetadata.glossary AS g
+SET (label_de, description_de) = (t.label_de, t.description_de)
+FROM pgmetadata.t_glossary AS t
+WHERE g.field = t.field AND g.code = t.code;
+
+DROP TABLE pgmetadata.t_glossary;
+
+
```

Comment:
```suggestion
```

As version 1.2.0 is already out, this should be removed.
It's isolated in a commit in case you need it.

Even for CI, it's make sure that the migration 1.2.3 is self contained. (and not using this migration 1.2.0)

Reply:
Should I move those changes into the upgrade_1.3.0 im preparing now? Otherwise, users who are on a version before 1.2.0 and are going to update to 1.3.0 might miss those changes. (See file processing/upgrade.py, lines 154–170 `for sf in sql_files:`)

Reply:
Yes, all changes to the database must go in a migration which is not released yet, that's why everything must go in a migration greater than 1.2.0.

https://github.com/3liz/qgis-pgmetadata-plugin/blob/master/pg_metadata/processing/database/upgrade.py#L151C42-L151C60 <= The current version of the database is used, so we have a list of migrations to apply. Someone having database version 1.0.0 for instance will launch, in the correct order, 3 migrations according to `available_migrations` : https://github.com/3liz/qgis-pgmetadata-plugin/tree/master/pg_metadata/install/sql/upgrade

Reply:
But haven’t version 1.2.0 – 1.2.2 already been released? (https://docs.3liz.org/qgis-pgmetadata-plugin/changelog/)

Reply:
I have renamed the former upgrade_1.2.3.sql to 1.3.0 and left 1.2.0 and 1.2.2 (no sql for 1.2.1) untouched, so I guess we’re on the same page now.
After you have merged this PR, I can update my PR #139.

Reply:
Sorry, there is still the file 1.2.0 modified here.
Look at your PR : https://github.com/3liz/qgis-pgmetadata-plugin/pull/103/files

Reply:
Ok, now I get it. I always thought you wanted me to remove the *whole* file, which confused me because it already was released. Now I realised that my changes have been applied to the released version of the file because the PR is so extremely old. (Originally, the PR would have introduced the file in the first place, as I have now seen in the very first message here…)
I’ll clean it up tomorrow and then the PR will hopefully be up to date.

Reply:
Indeed, we do not want to remove the old migration 🙂
Just edit a new migration file which has been released yet.

Thanks :)